ingest/mappings/dataset.json

```diff
@@ -0,0 +1,208 @@
+{
```

Comment:
When I grab the mapping from elasticsearch for dataset:

curl -XGET 'http://localhost:49200/dco/dataset/_mapping?pretty'

It is different from what's shown here. Thinks like the authrs.name.type is string in elastic search but in this mapping file it's multi_field. Some where the index is marked analyzed or non_analyzed are reversed in elasticsearch. Not sure this matters, but probably need to look into it.